.github/pull_request_template.md

```diff
@@ -12,6 +12,7 @@
 ## Checklist before requesting a review:
 - [ ] My code follows the guidelines of this project
 - [ ] I  performed a self-review of my code changes
+- [ ] I have added my changes to the change log (`README.md`)
```

Comment:
does this mean to mention `[HISTORY.rst](https://github.com/MAAP-Project/stac_ipyleaflet/blob/main/HISTORY.rst)` instead of the README?